packages/yoroi-extension/app/api/ergo/lib/transactions/utxoTransaction.js

```diff
@@ -454,7 +455,7 @@ export function signTransaction(request: {|
       RustModule.SigmaRust.ErgoBoxes.from_boxes_json([]), // TODO: not supported by sigma-rust
     );
 
-  const json = signedTx.to_json();
+  const json = JSONBigInt.parse(signedTx.to_json());
```

Comment:
@anon-br , if I remember correctly previously used version of the library were returning output values and asset values as strings. At least this is the contract for the dapp-connector API. So now that you are parsing JSON with the BigInt library - you might have to manually convert those specific fields to strings as they now will be parsed as numbers or BigNumbers

Reply:
@anon-br , yeah, if you check the return type of this function: `SignedRequest`

The fields `.outputs.*value` and `.outputs.*assets.*amount` must be returned as strings:

![image](https://user-images.githubusercontent.com/5585355/133499919-7ec9cb2a-c549-4102-a53f-6a31596114f5.png)

But now that you are using `JSONBigInt` - they will be parsed as `number | BigNumber` so in the return object you must explicitly convert it to `String(v)`

Reply:
Hmn gotcha